src/components/Contacts/ContactsList/ContactsList.tsx

```diff
@@ -22,7 +23,11 @@ export const ContactsList: React.FC = () => {
     <InfiniteList
       loading={loading}
       data={data?.contacts?.nodes ?? []}
-      style={{ height: `calc(100vh - ${navBarHeight} - ${headerHeight})` }}
+      style={{
```

Comment:
Interesting. I can get to the bottom without this change. In fact, adding those 8px causes double-scrolling for me. What do you think about pulling that change and putting it in a separate PR when we can figure out why it's behaving differently on different screens?

https://github.com/user-attachments/assets/f7ac383c-0dbb-4eb7-9e66-ab14f36a604f

Reply:
Yeah, I'll pull it out of the PR.

Reply:
For reference, here is what I get on the contact list

https://github.com/user-attachments/assets/27c263d1-5ab1-4120-bf44-66576373fb30